great_expectations/render/renderer/renderer.py

```diff
@@ -37,6 +38,10 @@ def inner_func(*args: P.args, **kwargs: P.kwargs):
 class Renderer:
     """A convenience class to provide an explicit mechanism to instantiate any Renderer."""
 
+    @override
+    def __eq__(self, other: object) -> bool:
+        return type(self) is type(other)
```

Comment:
Is this check good enough for renderers? We don't need to check anything inside them?

Reply:
Chatted offline. Renderers have so state so this is ok for equality.